### ISSUE TITLE
Include lantern-mobile Java code in Travis build

### ISSUE THREAD

**Issue by oxtoacart** [getlantern/lantern#4118]

**Comment by myleshorton** [comment]
Espresso looks super interesting for this:

https://google.github.io/android-testing-support-library/docs/espresso/

**Comment by atavism** [comment]
Yeah, Espresso is really nice. I started using it on the pro branch already. Looks like it's possible to always run those tests via Travis too
